Example/Unit Tests/AnimationSnapshotTests.swift

```diff
@@ -142,6 +160,18 @@ final class AnimationSnapshotTests: FBSnapshotTestCase {
         SnapshotVerify(animation: animation, on: view)
     }
 
+    func testAnimationWithNonViewElementSnapshotGIF() {
+        let view = View(frame: .init(x: 0, y: 0, width: 200, height: 40))
+
+        let element = Proxy(view: view)
+
+        var animation = Animation<Proxy>()
+        animation.addKeyframe(for: \.animatableViewTransform, at: 0, value: .identity)
+        animation.addKeyframe(for: \.animatableViewTransform, at: 1, value: .init(translationX: 160, y: 0))
+
+        SnapshotVerify(animation: animation, on: element, using: view)
+    }
```

Comment:
Is the difference between gif and the previous test, the fact that the former has interpolated values and the latter doesn't?

Reply:
The GIF one lets you generate an animated GIF of the entire animation, recorded at the specific FPS. This is usually the best way to snapshot your animations.

The other one lets you take a snapshot of a single frame of the animation, at the specified relative timestamp. This is helpful if you're trying to reduce the size of the snapshots, especially if you have a long running animation that would create a very large GIF.